tests/test_views.py

```diff
@@ -287,7 +287,7 @@ def test_env_per_page(bukudb, app, client, total, per_page, pages, last_page):
 
     response = client.get('/bookmark/last-page', follow_redirects=True)
     dom = assert_response(response, '/bookmark/', args={'page': str(pages - 1)})
-    cells = dom.xpath(f'//td{xpath_cls("col-Entry")}')
+    cells = dom.xpath(f'//td{xpath_cls("col-entry")}')
```

Comment:
Renamed the column key to lowercase since it's displayed using the original translation string for rendering anyway